.github/workflows/postgres.yaml

```diff
@@ -9,9 +9,9 @@ jobs:
   postgres:
     strategy:
       matrix:
-        go: [1.16]
+        go: [1.15]
```

Comment:
we run with golang 1.16.7 in production

Reply:
Yes, I think, It was mistakenly changed becuase of act

Reply:
Im trying 1.15 to see if the problem happens on 1.15.